app/assets/stylesheets/petitions/_forms.scss

```diff
@@ -38,16 +43,13 @@ select.form-control {
 
 .character-count {
   color: $secondary-text-colour;
-  text-align: right;
+  float: right;
```

Comment:
This makes it look pretty ugly on mobile without JS. We OK with that?

![screen shot 2015-07-07 at 11 15 09](https://cloud.githubusercontent.com/assets/1836301/8544025/891d7ed6-2499-11e5-8061-d1751bc31b69.png)